packages/api/cms-api/src/user-permissions/user-permissions.service.ts

```diff
@@ -178,6 +207,7 @@ export class UserPermissionsService {
         return {
             ...user,
             permissions,
+            impersonated: !!impersonatedUser,
         };
```

Comment:
We need the impersonated and the authenticated user in the access log. Currently, we only get the impersonated user. 

I think we can easily do this by adding an `authenticatedUser` field to the CurrentUser containing the actual user